src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -158,10 +161,12 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
                     pendingAction={item.pendingAction}
                     onToggle={item.action}
                     disabled={item.disabled}
+                    errors={item.errors}
+                    onCloseError={() => Policy.clearPolicyErrorField(policy?.id ?? '', 'areConnectionsEnabled')}
```

Comment:
```suggestion
                    onCloseError={item.onCloseError}
```
Move the function to integrateItems.